DatadogSessionReplay/Tests/Processor/Flattening/NodesFlattenerTests.swift

```diff
@@ -10,51 +10,21 @@ import Datadog
 @testable import TestUtilities
 
 class NodesFlattenerTests: XCTestCase {
-    /*
-          R
-         / \
-        I1  V1
-    */
-    func testFlattenNodes_withInvisibleNode() {
```

Comment:
Invisible nodes are no longer passed to `NodesFlattener` - so this test doesn't make sense anymore 💡